addons/mail/static/tests/qunit_suite_tests/chatter_tests.js

```diff
@@ -93,6 +93,57 @@ QUnit.test('list activity widget with activities', async function (assert) {
     assert.verifySteps(['/web/dataset/call_kw/res.users/web_search_read']);
 });
 
+QUnit.test('list activity widget with activities, two pages, mark done', async function (assert) {
+    const pyEnv = await startServer();
+    const mailActivityTypeId = pyEnv['mail.activity.type'].create({});
+    const mailActivityId = pyEnv['mail.activity'].create({
+        display_name: "Meet FP",
+        date_deadline: moment().add(1, 'day').format("YYYY-MM-DD"), // tomorrow
+        can_write: true,
+        state: "planned",
+        user_id: pyEnv.currentUserId,
+        create_uid: pyEnv.currentUserId,
+        activity_type_id: mailActivityTypeId,
+    });
```

Comment:
Probably worth patching date to middle of the day & week, to make sure this test doesn't fail close to midnights and week-ends :)

Reply:
fair enough